src/NServiceBus.AmazonSQS/MessagePump.cs

```diff
@@ -285,12 +282,60 @@ await onMessage(messageContext)
             }
         }
 
+        async Task MovePoisonMessageToErrorQueue(Message message)
+        {
+            try
+            {
+                await sqsClient.SendMessageAsync(new SendMessageRequest
+                {
+                    QueueUrl = errorQueueUrl,
+                    MessageBody = message.Body
+                }).ConfigureAwait(false);
+                // The MessageAttributes on message are read-only attributes provided by SQS
+                // and can't be re-sent. Unfortunately all the SQS metadata 
+                // such as SentTimestamp is reset with this send.    
```

Comment:
@SimonCropp I'm not a transport expert at all, do we need to set the FailedQ header here?